modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition_manager.py

```diff
@@ -107,7 +107,7 @@ def get_objects_from_partitions(cls, partitions):
         list
             The objects wrapped by `partitions`.
         """
-        return ray.get([partition._data for partition in partitions])
+        return ray.get([partition.list_of_blocks[0] for partition in partitions])
```

Comment:
Ditto from dask comment-- we should assert that each `list_of_blocks` has length 1 and put that assumption in the docstring.